achievements/contribubit.achievement.js

```diff
@@ -0,0 +1,33 @@
+var _ = require('lodash');
+var console = process.console;
+
+var TheGodfatherConsigliere = {
+	name: 'TheGodfatherConsigliere',
+	check: function(pullRequest, shall) {
+		if (contributedToKibibit(pullRequest)) {
+			//console.log("TheDunaevsky owns this...");
+			var achievement = {
+				avatar : 'https://worldfilmgeek.files.wordpress.com/2016/12/thegodfatherstill.jpg',
```

Comment:
change avatar to the new system. notice that all avatars now sit in [public/images/achievements/](https://github.com/Kibibit/achievibit/tree/master/public/images/achievements)

check out all other achievements for what to put in the `avatar` field